HIP/hip-904.md

```diff
@@ -768,9 +865,7 @@ Scenario 1: If Alice airdrops with the following transfer breakout
   association slots
 - NFT serials 3 & 4 of T4 to Carol, where Carol is not associated with T4 but has available
   association slots
-- 10 FTs of token T5 to Dave, where Dave is not associated with T5 and has no available
-  association slots
-- 5 FTs of token T5 to Dave, where Dave is not associated with T5 and has no available
+- 15 FTs of token T5 to Dave, where Dave is not associated with T5 and has no available
```

Comment:
I believe the split was deliberate to show the consolidation of fungible pending air drops.

Reply:
Ahh, I see.  We still need an example to refer to in other text for the intended scenario.  Might be worth ensuring that is present.